File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        n=len(nums)
        r=[]
        for i in range(n):
            for j in range(i+1,n):
                if nums[i]+nums[j]==target:
                    r.append(i)
                    r.append(j)
                    return r

File: 0001-two-sum/0001-two-sum.java
import java.util.*;
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer, Integer> map = new HashMap<>();
        int[] result = new int[2];
        for(int i = 0;i<nums.length;i++){
            int res = target-nums[i];
            if(map.containsKey(nums[i])){
                result = new int[]{map.get(nums[i]),i};
                return result;
            }
            else{
                map.put(res,i);
            }
        }
        return result;
    }
}